ivadomed/utils.py

```diff
@@ -154,3 +153,14 @@ def threshold_predictions(predictions, thr=0.5):
     return thresholded_preds
 
 
+def cuda(input_var):
+    """
+    This function sends input_var to GPU.
+    :param input_var: either a tensor or a list of tensors
+    :return: same as input_var
+    """
+
+    if isinstance(input_var, list):
+        return [t.cuda() for t in input_var]
+    else:
+        return input_var.cuda()
```

Comment:
Need to add this function otherwise `.cuda()` returns an error when `input_samples` is a list of tensors.